src/app/api/file/route.ts

```diff
@@ -0,0 +1,132 @@
+import { NextResponse } from "next/server";
+import { File, FileResponse } from "./route.schema";
+import { prisma } from "@server/db/client";
+import { withSession } from "@server/decorator";
+import { google } from "googleapis";
+
+export const POST = withSession(async (request) => {
+  console.log("IN POST");
+
+  try {
+    console.log("STARTING TRY");
+    const auth = new google.auth.OAuth2({
+      clientId: process.env.GOOGLE_CLIENT_ID,
```

Comment:
Should use `env` variable that is exported from `./src/env/server.mjs` for type safety

Reply:
Perfect! Fixed!